Dockerfile-alpine.template

```diff
@@ -33,7 +33,7 @@ RUN addgroup -g 1000 node \
         libgcc \
         linux-headers \
         make \
-        python \
+        python3 \
```

Comment:
Maybe `${PYTHON_VERSION}` and do the replacement in the update. (Just kind of repeating what you said you were working on :))
You could fuzzy match in here for the replacement https://github.com/nodejs/docker-node/blob/3a5c226f511b1450eff15600aa4f877b272eeefd/update.sh#L163-L169

Reply:
Ah yes that is a better way that how I initially did it. Let me do it that way.

Reply:
How is it looking now?